riak_test/yz_pb.erl

```diff
@@ -244,3 +247,28 @@ confirm_stored_fields(Cluster) ->
     ?assertEqual(Index, proplists:get_value(<<"_yz_rt">>, Fields)),
     ?assertEqual(<<"b1">>, proplists:get_value(<<"_yz_rb">>, Fields)),
     riakc_pb_socket:stop(Pid).
+
+confirm_search_to_test_max_score_defaults(Cluster) ->
+    Index = <<"basic_for_max_score">>,
+    Bucket = {Index, <<"b1">>},
+    create_index(Cluster, Index, Index),
+    lager:info("confirm_search_to_test_max_score_defaults", [Bucket]),
+    Body = <<"{\"age_i\":5}">>,
+    Params = [{sort, <<"age_i asc">>}],
+    HP = select_random(host_entries(rt:connection_info(Cluster))),
+    URL = bucket_url(HP, Bucket, "multivalued"),
+    lager:info("Storing to bucket ~s", [URL]),
+    {Host, Port} = HP,
+    %% populate a value
+    {ok, "204", _, _} = ibrowse:send_req(URL, [?CT_JSON], put, Body),
+    %% Sleep for soft commit
+    timer:sleep(1100),
+    Search = <<"age_i:5">>,
+    {ok, Pid} = riakc_pb_socket:start_link(Host, (Port-1)),
+    {ok,{search_results,[{Index,Fields}],Score,_Found}} =
+        riakc_pb_socket:search(Pid, Index, Search, Params),
+    ?assertNotEqual(Score, []),
+    ?assertNotEqual(Score, 0),
```

Comment:
We can't determine the extra score from solr, but I'll push a line of cleanup right now actually :).